src/peer_manager.rs

```diff
@@ -271,170 +174,6 @@ impl PeerManager {
         self.insert_peer(pub_id, conn_info.into());
     }
 
-    /// Return true if received CandidateInfo
-    #[cfg(all(test, feature = "mock_parsec"))]
-    pub fn has_candidate_info(&self) -> bool {
-        if let Candidate::ResourceProof { .. } = self.candidate {
-            true
-        } else {
-            false
-        }
-    }
-
-    /// Our section decided that the candidate should be selected next.
-    /// Store start time so we can detect when candidate expires.
-    pub fn accept_as_candidate(&mut self) {
-        self.candidate = Candidate::AcceptedForResourceProof {
-            res_proof_start: Instant::now(),
-            expired_once: false,
-        };
-    }
-
-    /// Verifies proof of resource.  If the response is not the current candidate, or if it fails
-    /// validation, returns `Err`.  Otherwise returns the target size, difficulty and the time
-    /// elapsed since the candidate was inserted.
-    pub fn verify_candidate(
```

Comment:
Can you clarify in PR comment that in addition to not resource proof, we are also not checking that the node is in the correct interval.

Are we doing no checks at all, what if it comes with a public id that is not part of our section?
=> answering my own question: if !target_interval.contains(new_pub_id.name()) {